### PR TITLE
v8.1.0 - port over f-utils scss.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added
- f-utils `functions`, `helpers` & `mixins` in order to depreciate `f-utils` in favour of `fozzie`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
     "node": ">=10.0.0"
```
Comment (ashleynolan) [inline package.json:33]: Any chance we can update this node version as well? Looks a bit out of date

File: src/scss/tools/_index.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
 // =================================
-// Tools
+// Fozzie SCSS Utilities
```
Comment (ashleynolan) [inline src/scss/tools/_index.scss:2]: I'd keep this as tools, as it's the index of that folder

```suggestion
// Tools
```

File: src/scss/_dependencies.scss [hunk 3]
```diff
@@ -26,7 +26,7 @@
 @import '@justeat/pie-design-tokens/dist/jet';
 
 @import 'settings/variables';
-@import '@justeat/f-utils/src/scss/helpers/breakpoints'; // breakpoint helper – https://github.com/justeat/f-utils/blob/96508ae19809c4b8b83cebb55b9d74728d01c1a5/src/scss/helpers/_breakpoints.scss
+@import 'tools/helpers/breakpoints'; // breakpoint helper
```
Comment (ashleynolan) [inline src/scss/_dependencies.scss:29]: Would be good to remove this by default and add it into the `_templates` as I think we only really use this on some of our older applications.

Would make this change a breaking change then though, so maybe could you add a `TODO` to this line that we'll remove/move it as part of the next major version bump?

**Review by ashleynolan — COMMENTED** [review]
Looks good, just a couple of comments

**Comment by kevinrodrigues** [comment]
> Looks good, just a couple of comments

Thanks. Have added the suggested changes now 👍

**Review by ashleynolan — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**